New package: frontier-client

iarspider opened the pull request:
Depends on #21372

iarspider committed:
New package: frontier-client

skosukhin committed:
autoconf: fix v2.70 and patching procedure of bin/autom4te.in  (#21255)

This PR adds:

1.  A patch that fixes a bug in version 2.70 
    (will be fixed upstream in the next release: https://savannah.gnu.org/support/?110396).
2. A fix for the way we patch shebang in bin/autom4te.in. 

For 2, we need to keep the original modification timestamp of the file. 
Otherwise, we either get an empty man page for autom4te (versions 2.69 and before)
or a failure at the build time (versions 2.70 and after). 

The difference has to do with the update of the missing script: https://git.savannah.gnu.org/cgit/automake.git/commit/lib/missing?id=a22717dffe37f30ef2ad2c355b68c9b3b5e4b8c7

It will take time until developers of Autotools-based packages adjust their scripts 
to the new version, therefore, 2.69 is marked as preferred.

iarspider committed:
Fix recipe

iarspider committed:
umesimd: add new package (#21370)

bryanherman committed:
openmc: add v0.11.0, v0.12.0  (#21349)

Also added new variants to fine tune the build

bryanherman committed:
New package: py-pytest-openfiles (#21339)

* added package py-pytest-openfiles

* remove wheel dep

bryanherman committed:
New package: py-pytest-doctestplus (#21335)

* added package py-pytest-doctestplus

* setuptools required for run

tldahlgren reviewed (changes requested):
LGTM.  Only change requested is to rebase after dependent PR is merged to ensure tests can pass.

adamjstewart committed:
py-matplotlib: add v3.3.4 (#21378)

aahmed-se committed:
New package: libpulsar (#21369)

t-nojiri committed:
openmx: Fix for aarch64 (#21367)

adamjstewart committed:
Make git fetching even quieter (for git v1.7.2+)(#21013)

iarspider committed:
pacparser: new package at v1.3.7 (#21372)


Co-authored-by: Massimiliano Culpo <massimiliano.culpo@gmail.com>

dorton21 committed:
New Package r-fpCompare@0.2.3 (#21380)

alalazo committed:
perl: add v5.32.1 (#21375)

Make it such that BigSur is seen as 10.16 rather than 11.1, since the latter causes troubles with Perl configuration

danlipsa committed:
Use utf8 for latex build. (#21360)

Patch provided by @Billae

Avoid the following error:
  File "/home/danlipsa/projects/spack/lib/spack/llnl/util/tty/log.py", line 768, in _writer_daemon
    line = _retry(in_pipe.readline)()
  File "/home/danlipsa/projects/spack/lib/spack/llnl/util/tty/log.py", line 830, in wrapped
    return function(*args, **kwargs)
  File "/usr/lib/python3.8/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x97 in position 220: invalid start byte

cyrush committed:
add new package: bloaty (#21355)

adamjstewart committed:
py-graphviz: add v0.8.4 (#21383)

t-nojiri committed:
swarm: add v3.0.0 (#21389)

t-nojiri committed:
wtdbg2: enable build for aarch64 (#21390)

Use sse2neon to maintain the same API calls for aarch64

t-nojiri committed:
py-cheetah3: add new package at v3.2.6 (#21387)

tldahlgren commented:
Hmm.  I don't think the merge conflict resolution is quite right since it *appears* to be adding you as a contributor to all of the other PRs.  Or am I missing something?

iarspider commented:
Weird... Let me redo the MR.
